v1-1/learn/how-to-deploy-and-run-ballerina-programs.md

```diff
@@ -211,9 +211,9 @@ service helloWorld on new http:Listener(9090) {
     resource function sayHello (http:Caller caller, http:Request request) {
         http:Response response = new;
         response.setTextPayload("Hello, World! \n");
-        var result = caller -> respond(response);
-        if (result is error) {
-            log:printError("Error sending response", result);
+        var responseResult = caller -> respond(response);

```

Comment:
Any specific reason to change this to `responseResult` instead of `result`?  We are using `result` in a similar context for other examples and BBEs. IMO, the variable name is lengthier and add not much value as both words are bit similar. Also since you are changing a few formatting issues here, better to fix all the places. There are more formatting issues in the sample.

Reply:
These issues will be fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/201.